env-info/src/main/java/io/quarkus/ts/envinfo/OsJvmEnvInfoLogger.java

```diff
@@ -53,4 +63,15 @@ private static String getJavaVersionCmdOutput() throws InterruptedException, IOE
         return new BufferedReader(new InputStreamReader(pr.getInputStream())).lines().reduce((first, second) -> second)
                 .orElse(null);
     }
+
+    private static List<String> getMvnVersionCmdOutput() {
+        final String cmd = System.getProperty("maven.home") + File.separator + "bin" + File.separator + "mvn --version";
+        try {
```

Comment:
I would prefer letting the exceptions show themselves.

Reply:
I did this on purpose, the exception breaks the run of mvn command / whole TS.

Reply:
It is IMO correct to break the whole run when the mvn command fails.

Reply:
> It is IMO correct to break the whole run when the mvn command fails.
 
Definitely not, the main goal is to run the TS. 
Getting env details is not the main goal of TS execution.
TS shouldn't stop because the secondary task didn't finish properly.

The command fails only if `System.getProperty("maven.home")` gives undefined / non-relevant value.

Reply:
"java.home" property is defined thanks to JDK so it's fine to fail if `getJavaVersionCmdOutput` doesn't succeed.

"maven.home" property is not defined thanks to JDK and I didn't find any guarantee for the availability. So it's better to return empty result than stop the world.

Reply:
Ok. I think it would be better to have the environment set up in such a way that would guarantee the consistent availability of Maven (after all, we do always run a maven build), but let's not block this PR because of it.